core/server/master/pom.xml

```diff
@@ -136,6 +141,12 @@
       <groupId>org.apache.httpcomponents</groupId>
       <artifactId>httpclient</artifactId>
     </dependency>
+      <dependency>
```

Comment:
It looks like the indentation here is not aligned with the previous line.

Reply:
good catch. fixed.